src/fs.js

```diff
@@ -55,17 +46,6 @@ Fs.prototype = {
         if (lookups.hasOwnProperty(b)) {
           var checkDir = path.join(relativeTo, new Array(a).join('../'), b, mod);
           var packageFile = path.join(checkDir, lookups[b]);
```

Comment:
It may still be worth sorting your iteration over `lookups` to give well defined behaviour in case someone does have a package in both `bower_components` and `node_modules` and it "works for them" for a while 'cause they got lucky as to the iteration order.